tests/integration/components/paper-toolbar-test.js

```diff
@@ -0,0 +1,20 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('paper-toolbar', 'Integration | Component | paper toolbar', {
+  integration: true
+});
+
+test('should bind class names correctly', function(assert) {
```

Comment:
let's test for md-tall class only.
We'll probably add md-default-theme to another place in the code base soon.